warehouse/query-core/src/main/java/datawave/query/jexl/HitListArithmetic.java

```diff
@@ -221,7 +222,7 @@ public boolean lessThan(final Object left, final Object right) {
             for (final Object o : set) {
                 Object normalizedO = ValueTuple.getNormalizedValue(o);
                 Object normalizedRight = ValueTuple.getNormalizedValue(right);
-                if (compare(normalizedO, normalizedRight, LESS_THAN) < 0) {
+                if (compare(normalizedO, normalizedRight, JexlOperator.LT) < 0) {
```

Comment:
I don't think the string constants on line 26 are needed anymore

Reply:
+1

Reply:
still not resolved